### PR TITLE
brownfield: Documentation

### PR DESCRIPTION
**Pull request by delqn**
The last piece of brownfield deployment is adding the documentation. 🍒 🍨

### CODE REVIEW

File: docs/setup/install-existing.md [hunk 1]
```diff
@@ -107,22 +149,161 @@ The [aad-pod-identity](https://github.com/Azure/aad-pod-identity) gives a clean
...
+Enabling `brownfield.enabled=true` and `helm` installing AGIC, would result in the creation of a `prohibit-all-targets` object in your AKS.
+
+View the contents of the object:
+```bash
+kubectl get AzureIngressProhibitedTarget prohibit-all-targets -o yaml
+```
+
+You can broaden the permissions AGIC has:
+1. Create a new `AzureIngressProhibitedTarget` with your specific setup:
+    ```bash
```
Comment (akshaysngupta) [inline docs/setup/install-existing.md]: Shouldn't this be enough ?
```bash
helm upgrade \
          --recreate-pods \
          --version 0.8.0 \
          -f .dev/helm-config.yaml \
          --set appgw.shared=true
          ingress-azure application-gateway-kubernetes-ingress/ingress-azure
```

Comment (delqn) [reply docs/setup/install-existing.md]: It turns out - not.  I ran into this:
```
[debug] Created tunnel using local port: '61149'

[debug] SERVER: "127.0.0.1:61149"

UPGRADE FAILED
ROLLING BACK
Error: failed to create resource: the server could not find the requested resource (post azureingressprohibitedtargets.appgw.ingress.k8s.io)
Error: UPGRADE FAILED: failed to create resource: the server could not find the requested resource (post azureingressprohibitedtargets.appgw.ingress.k8s.io)
```
Perhaps this is a matter of properly configuring the helm upgrade scenario.

Comment (delqn) [reply docs/setup/install-existing.md]: And after I create the CRD - `helm` then creates `prohibit-all-targets` as expected.

File: docs/setup/install-existing.md [hunk 2]
```diff
@@ -107,22 +149,161 @@ The [aad-pod-identity](https://github.com/Azure/aad-pod-identity) gives a clean
...
+    helm install ./helm/ingress-azure \
+         --name ingress-azure \
+         --version 0.7.1 \
+         --namespace default \
+         --debug \
+         --set appgw.name=applicationgatewayABCD \
+         --set appgw.resourceGroup=your-resource-group \
+         --set appgw.subscriptionId=subscription-uuid \
+         --set armAuth.type=servicePrincipal \
+         --set armAuth.secretJSON=$(az ad sp create-for-rbac --subscription <subscription-uuid> --sdk-auth | base64 -w0) \
```
Comment (akshaysngupta) [inline docs/setup/install-existing.md]: omit the version

File: docs/setup/install-existing.md [hunk 3]
```diff
@@ -107,22 +149,161 @@ The [aad-pod-identity](https://github.com/Azure/aad-pod-identity) gives a clean
...
+## Hybrid Environment
+AGIC will assume full ownership of the AGIC configuration. What if we'd like to manually add a listener and a backend?
+We have the option to constrain AGIC, so it controls only a subset of the App Gateway's properties.
+
+### Scenario
+Let's take an imaginary App Gatway, which manages traffic for 2 web sites:
+  - prod.contoso.com
+  - dev.contoso.com
+
+We need to:
```
Comment (akshaysngupta) [inline docs/setup/install-existing.md]: should we mention what version support sthis feature ?

Comment (delqn) [reply docs/setup/install-existing.md]: Not a bad idea - adding...

**Pull request merged**